### PR TITLE
Map PlanetEvent MaxHealth

### LINKED ISSUE #43: [BUG] planet-events does not give event's maxHealth for planet
### Link to faulty endpoint

https://helldivers-2-dotnet.fly.dev/api/v1/planet-events

### The name of your client/application

Mozilla / Swagger_UI Page

### Request information

Mozilla / Swagger_UI Page

### Steps to reproduce

Send get request to https://helldivers-2-dotnet.fly.dev/api/v1/planet-events to get planet event info

### What is expected?

event maxHealth to be depicted in the event key

### What is actually happening?

```json
maxHealth is missing from event key
```

### System Info

```shell
Windows 10
Mozilla FF
https://helldivers-2-dotnet.fly.dev/swagger-ui.html
```

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by lambstream**
Closes #43

### CODE REVIEW

**Review by chatterchats — COMMENTED** [review]
Looks good to me! @dealloc I'll leave final to you

File: src/Helldivers-2-Models/ArrowHead/Status/PlanetEvent.cs [hunk 1]
```diff
@@ -9,7 +9,8 @@ namespace Helldivers.Models.ArrowHead.Status;
 /// <param name="PlanetIndex">The index of the planet.</param>
 /// <param name="EventType">A numerical identifier that indicates what type of event this is.</param>
 /// <param name="Race">The identifier of the faction that owns the planet currently.</param>
-/// <param name="Health">The current health of the planet.</param>
+/// <param name="Health">The current health of the event.</param>
```
Comment (dealloc) [inline src/Helldivers-2-Models/ArrowHead/Status/PlanetEvent.cs:12]: The event is about a planet, so wasn't the original docs correct?

Comment (lambstream) [reply src/Helldivers-2-Models/ArrowHead/Status/PlanetEvent.cs:12]: Seeming as the health differs from the planet health i think the health was specific to the event?

Comment (dealloc) [reply src/Helldivers-2-Models/ArrowHead/Status/PlanetEvent.cs:12]: makes sense

**Review by dealloc — APPROVED** [review]

**Pull request merged**